client/src/pages/about.js

```diff
@@ -31,8 +16,8 @@ const AboutPage = () => {
         <Row>
           <Col
             className='questions'
-            md={6}
-            mdOffset={3}
+            md={8}
+            mdOffset={2}
```

Comment:
```suggestion
            mdOffset={3}
```

Please revert this back to what it was.